astro/src/content/docs/release-notes/index.mdx

```diff
@@ -374,11 +374,9 @@ Please be sure to read the notes in the **Changed** section before upgrading.
 
 ### Fixed
 * Update `fusionauth/java-http` to the most recent version to pick up a bug fix.
-+
-This fixes a very low level HTTP server bug. In some rare cases, the HTTP response handler may not identity the end of the stream and effectively truncate the response body. It is difficult to say how how may affect your integration if you were to encounter it. If you were to make an API call with a large response body, it may be possible the response would not include a valid JSON object if the response is truncated. When this error occurs, the HTTP status code will be valid, but the response will be truncated or non-existent.
-+
-For additional detail see the linked commit in the linked GitHub issue.
 
+  This fixes a very low level HTTP server bug. In some rare cases, the HTTP response handler may not identity the end of the stream and effectively truncate the response body. It is difficult to say how how may affect your integration if you were to encounter it. If you were to make an API call with a large response body, it may be possible the response would not include a valid JSON object if the response is truncated. When this error occurs, the HTTP status code will be valid, but the response will be truncated or non-existent. \
```

Comment:
```suggestion
  This fixes a very low level HTTP server bug. In some rare cases, the HTTP response handler may not identify the end of the stream and effectively truncate the response body. It is difficult to say how how may affect your integration if you were to encounter it. If you were to make an API call with a large response body, it may be possible the response would not include a valid JSON object if the response is truncated. When this error occurs, the HTTP status code will be valid, but the response will be truncated or non-existent. \
```